core/capabilities/compute/monitoring.go

```diff
@@ -0,0 +1,10 @@
+package compute
+
+// Observability keys
+const (
+	wIDKey       = "workflowID"
+	eIDKey       = "workflowExecutionID"
+	wnKey        = "workflowName"
+	woIDKey      = "workflowOwner"
```

Comment:
Let's use full names here, those are very hard to read.

Reply:
Let's fix other places as well (@patrickhuie19 ?)

Reply:
I just fixed the other ones.